config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -30,7 +30,7 @@ presubmits:
         - --cluster=
         - --env=KUBE_PROXY_MODE=ipvs
         - --extract=local
-        - --gcp-node-image=ubuntu
+        - --gcp-node-image=gci
```

Comment:
@dims does this ring a bell? 
e2e flexvolumes tests and node images are related somehow?